README.md

```diff
@@ -33,6 +34,33 @@ As mentioned in the introduction, the CTB application models many recommended pr
 - It can run using [`npm`](https://www.npmjs.com/get-npm) on a \*nix OS or as a [Docker](https://docs.docker.com/install/) container
 - It includes [terraform scripts](https://github.com/cds-snc/cra-claim-tax-benefits/tree/master/scripts) for deploying either to Azure or AWS
 
+### Use of third-party services
+
+We use several third-party services for an improved development workflow and continuous security.
+
+- [GitHub](https://github.com/) is a cloud-based service to store our source code, as well as track changes and facilitate code reviews
+- [GitHub Actions](https://github.com/features/actions) is a CI/CD service that allows us to [test and deploy our code](https://github.com/cds-snc/cra-claim-tax-benefits/blob/master/.github/workflows/testBuildDeploy.yml) right from GitHub
```

Comment:
maybe add what the acronym stands for (I think most will know CI/CD, but doesn't hurt)